### ISSUE TITLE
Anchor to TabView or TabItem

### ISSUE THREAD

**Issue by t8n** [aheze/Popovers#58]
I'm trying to anchor a popover to a tab item, but I can't even get it to show.

I also tried on the TabView, but that doesn't seem to work either.

Can anyone offer any help? Is it possible?

**Comment by jforward5** [comment]
did you find a way?  I am also looking for a way to do this.

**Comment by t8n** [comment]
No. What I did instead was offset it high enough so it is definitely above the tab bar (with a gap) and then point down in the centre. Definitely not ideal...
